src/Equations/RunawayFluid.cpp

```diff
@@ -336,7 +337,16 @@ void RunawayFluid::CalculateGrowthRates(){
 
     for (len_t ir = 0; ir<this->nr; ir++){
         avalancheGrowthRate[ir] = n_tot[ir] * constPreFactor * criticalREMomentumInvSq[ir];
-        real_t pc = criticalREMomentum[ir]; 
+
+        // multiply by trapping correction factor
+        if (ava_trapping == OptionConstants::EQTERM_AVALANCHE_TRAPPING_CORRECTION_INCLUDE){
+
+            real_t eps = (this->rGrid->GetR(ir)) / (this->rGrid->GetR0());
+            real_t phi = 1/(1 + 1.46*sqrt(eps) + 1.72*eps);
```

Comment:
isn't this phi just a poor man's approximation to our rGrid->effectivePassingFraction which is the correct generalization to arbitrary geometry? @hoppe93